README.md

```diff
@@ -187,12 +224,11 @@ tests in the system. In such a case, you can speed up improve your change-the-te
 running selected Cucumber scenario(s) rather than all.
 
 To do that:
-- tag selected scenario(s) with a [custom tag][Gherkin tags], e.g. `@RUN`, placing it in the line directly preceding
-  the scenario,
-- modify class `AcceptanceTest`, replacing value of attribute `tags` in the by `@CucumberOptions` annotation with
-  your custom tag.
+- tag selected scenario(s) with a [custom tag][Gherkin tags] `@WIP`, placing it in the line directly preceding
```

Comment:
This probably means that the `~@WIP` tag should be removed from `AcceptanceTest` class, too.

The original idea behind `~@WIP` in `AcceptanceTest` was that `@WIP` could be used to flag scenarios that we (temporarily) _don't_ want to have executed (say, when we want to commit our in-flight changes but we don't want them to create noise by known failing tests breaking the build). The `@RUN` tag was meant to target scenarios that we want to run exclusively.

Since `@WIP` seems to have taken over the original `@RUN`'s role, the continuing presence of `~@WIP` in `AcceptanceTest` may cause confusion or even interfere with tests.

Reply:
Ah yes didn't know about the other WIP tag, have removed it